src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -146,17 +131,33 @@ C_FUNC(\Name\()_End):
 
         __PWTB_StackAlloc = __PWTB_TransitionBlock
 
+	// Make the stack operation compatible with GDB and LIBUNWIND
+	PROLOG_PUSH		"{r7, lr}"
```

Comment:
Alright. My next push will comply with that assumption while being atomic for `r7` update and branching.

Question: is it ok to destroy the `TransitionBlock` data structure at the moment of leaving at `EPILOG_WITH_TRANSITION_BLOCK_*` ?